docs/conceptual/node/tx-pool.md

```diff
@@ -3,41 +3,52 @@ title: Transaction Pool
 
-1) Checking if transactions are valid or invalid. For example, ensuring the account that issued the transaction has enough funds to pay the associated fees.
+
+1) Checking if transactions are valid. `validity` of the transaction is not hard wired to the transaction pool, but is defined by the runtime through the `validate_transaction` function ([code](https://github.com/paritytech/substrate/blob/20a9b15cdbed4bf962a4447e8bfb812f766f2fbc/frame/executive/src/lib.rs#L304)). Example validity checks are:
+
+* Transaction Index (nonce) correctness.
+* The account has enough funds to pay for the associated fees.
+* The signature is valid.
 
 2) If the transaction is valid, sorting the transactions into 2 transaction queues:
    - **Ready Queue** -  Contains transactions that can be included in a new pending block and must follow the *exact* order in the ready queue.
 
-   - **Future Queue** - Contains transactions that *seem* valid, but the full node has yet to see the remaining transactions it depends on. For example, *Transaction B* is in the future queue and can only be included in the block after *Transaction A* has been included. However, the node is yet to see *Transaction A* in its transaction pool.
+   - **Future Queue** - Contains transactions that *seem* valid so far, but the full node has yet to see the remaining transactions it depends on. For example, *Transaction B* is in the future queue and can only be included in a block after *Transaction A* has been included. However, the node is yet to see *Transaction A* in its transaction pool.
+
+Full nodes in the network all have their own local transaction pool and transaction queues. Therefore, full nodes may have a different view of the queues and transactions in the network.
 
-Full nodes in the network all have their own local transaction pool and transaction queues. Therefore, full nodes can implement different strategies on how they prioritise importing transactions.
+Dependent on the runtime, strict ordering may not be a requirement. This could enable full nodes to implement different strategies on how they prioritise propagating transactions and how they're included into blocks.
 
 ## Transaction Dependencies
 
-Transaction dependencies are defined by the `requires` and `provides` parameters in `ValidTransaction` to build a dependency graph of transactions. This allows the importation of transactions in a correct and linear order.
+Transaction [dependencies](https://substrate.dev/rustdocs/master/sp_runtime/transaction_validity/struct.ValidTransaction.html) are defined by the `requires` and `provides` parameters in `ValidTransaction` to build a dependency graph of transactions. Together with `priority`, this allows the pool to traverse the graph and produce a valid linear ordering of transactions to be propagated and included in the next block.
```

Comment:
Seems like the link should be on the `ValidTransaction` rather than "dependencies"  since that's the struct you're linking to